sfaira/data/dataloaders/loaders/d10_1038_s41586_018_0654_5/Mouse_Brain_2018_Smart-seq2_Tasic_001.yaml

```diff
@@ -0,0 +1,44 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
```

Comment:
You can leave this field empty if you only have a single file name here to reduce complexity!